app/controllers/user_controller.rb

```diff
@@ -0,0 +1,112 @@
+require "sinatra/base"
+require "sinatra/activerecord"
+require "bcrypt"
+require_relative "../models/user"
+require_relative "./application_controller"
+require_relative "../helpers/session_helper"
+
+class UserController < Sinatra::Base
+  include SessionHelper
+  enable :sessions
+  register Sinatra::ActiveRecordExtension
+  configure do
+    set :views, 'app/views'
+  end 
+
+  get "/sign_in" do 
+    if logged_in?
+      erb(:redirect_to_user_page)
+    else
+      erb(:sign_in)
+    end 
+  end 
+    
```

Comment:
It would be super helpful to have some comments with a little description of what something is doing or where to go for more information about certain methods